<robot name="robo_2d">

<link name="base">
    
    <visual name="">
        <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
        <geometry>
<<<<<<< HEAD
            <box size="0.2 0.15 0.025"/>
=======
            <box size="0.2 0.025 0.15"/>
>>>>>>> 5a3f4bc... Uploading the created package with URDf
        </geometry>
        <material name="">
            <color rgba="1.0 0.0 0.0 1.0"/>
            <texture filename=""/>
        </material>
    </visual>

</link>

<link name="link1">

    <visual name="">
<<<<<<< HEAD
        <origin xyz="-0.025 0.0 0.15" rpy="0.0 0.0 0.0"/>
        <geometry>
            <box size="0.25 0.15 0.5"/>
=======
        <origin xyz="-0.025 0.15 0.0" rpy="0.0 0.0 0.0"/>
        <geometry>
            <box size="0.25 0.5 0.15"/>
>>>>>>> 5a3f4bc... Uploading the created package with URDf
        </geometry>
        <material name="">
            <color rgba="1 1 1 1"/>
            <texture filename=""/>
        </material>
    </visual>

</link>

<link name="link2">

    <visual name="">
<<<<<<< HEAD
        <origin xyz="0.0 0.0 0.25" rpy="0.0 0.0 0.0"/>
        <geometry>
            <box size="0.25 0.15 0.5"/>
=======
        <origin xyz="0.0 0.25 0.0" rpy="0.0 0.0 0.0"/>
        <geometry>
            <box size="0.25 0.5 0.15"/>
>>>>>>> 5a3f4bc... Uploading the created package with URDf
        </geometry>
        <material name="">
            <color rgba="1 1 0.5 1"/>
            <texture filename=""/>
        </material>
    </visual>

</link>

<link name="endlink">
 
    <visual name="">
        <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
        <geometry>
            <box size="0.0 0.0 0.0"/>
        </geometry>
        <material name="">
            <color rgba="1.0 0.0 0.0 1.0"/>
            <texture filename=""/>
        </material>
    </visual>

</link>


<joint name="joint1" type="continuous">
<<<<<<< HEAD
    <origin xyz="0.025 0.0 0.1" rpy="0.0 0.0 0.0"/>
=======
    <origin xyz="0.025 0.1 0.0" rpy="0.0 0.0 0.0"/>
>>>>>>> 5a3f4bc... Uploading the created package with URDf
    <parent link="base"/>
    <child link="link1"/>
    <axis xyz="0.0 1 0.0"/>
 
</joint>

<joint name="joint2" type="continuous">
<<<<<<< HEAD
    <origin xyz="-0.025 0.0 0.4" rpy="0.0 0.0 0.0"/>
=======
    <origin xyz="-0.025 0.4 0.0" rpy="0.0 0.0 0.0"/>
>>>>>>> 5a3f4bc... Uploading the created package with URDf
    <parent link="link1"/>
    <child link="link2"/>
    <axis xyz="0.0 1 0.0"/>

</joint>

<joint name="end_effector" type="fixed">
    <origin xyz="0.0 0.0 0.0" rpy="0.0 0.0 0.0"/>
    <parent link="link2"/>
    <child link="endlink"/>
</joint>
<<<<<<< HEAD

</robot>

=======
    
</robot>
>>>>>>> 5a3f4bc... Uploading the created package with URDf
